src/components/RadioButtons/RadioButtons.jsx

```diff
@@ -109,6 +126,7 @@ class RadioButtons extends React.Component {
               onKeyDown={this.props.onKeyDown}
               value={optionValue}
               onChange={this.handleChange}
+              data-label={optionLabel}
```

Comment:
@cvalarida - I need to be able to access the option's label in the analytics event.  Got a smarter way than just jamming it into a `data-` attribute?

Reply:
There are a few ways you could get that text without using a `data-` attribute, yeah. Most of them are kinda brittle, but this fancy bit of tomfoolery in `handleChange` should do the trick and be fairly maintainable later on if we decide to change up the structure of the HTML this component outputs:
```js
domEvent.target.parentElement.querySelector(`label[for="${domEvent.target.id}"`).innerText;
```
![image](https://user-images.githubusercontent.com/12970166/117225013-2fae1600-adc6-11eb-8981-83deeb3d1981.png)